locale/ar/docs/meta/topics/dependencies.md

```diff
@@ -0,0 +1,78 @@
+---
+title: الإعتمادات
+layout: docs.hbs
+---
+
+# الإعتمادات
+
+هناك العديد من الإعتمادات التي يعتمد عليها Node.js للعمل بالطريقة التي يعمل بها.
+
+- [المكتبات](#libraries)
+  - [V8](#v8)
+  - [libuv](#libuv)
+  - [http-parser](#http-parser)
+  - [c-ares](#c-ares)
+  - [OpenSSL](#openssl)
+  - [zlib](#zlib)
+- [الأدوات](#tools)
+  - [npm](#npm)
+  - [gyp](#gyp)
+  - [gtest](#gtest)
+
+## <!--libraries-->المكتبات
+
+### V8
+
+تزود مكتبة V8 <span dir='ltr'> Node.js </span> بمحرك JavaScript ،الذي يتحكم به Node.js عبر واجهة برمجة التطبيقات V8 C++. يتم الإشراف على V8 بواسطة Google ، للاستخدام في Chrome.
+
+- [التوثيق](https://v8docs.nodesource.com/)
+
+### libuv
+
+إعتمادية مهمة الأخرى هي libuv ، وهي مكتبة C تُستخدم لتجريد عمليات الإدخال / الإخراج غير المحظورة إلى واجهة متناسقة عبر جميع الأنظمة الأساسية المدعومة. إنه يوفر آليات لمعالجة ملفات النظام ، DNS ، الشبكة ، العمليات الفرعية ، التصفية ، معالجة الإشارات ، الاقتراع والتدفق. يتضمن أيضًا تجمع مؤشرات ترابط لإلغاء تحميل العمل لبعض الأشياء التي لا يمكن القيام بها بشكل غير متزامن على مستوى نظام التشغيل.
+
+- [التوثيق](http://docs.libuv.org/)
+
+### http-parser
+
+تتم معالجة تحليل HTTP بواسطة مكتبة C خفيفة الوزن تسمى http-parser. تم تصميمه بحيث لا يقوم بإجراء أي نداءات نظام (syscalls) أو عمليات تخصيص ( allocations ) ، لذلك يحتوي على مساحة ذاكرة صغيرة جدًا لكل طلب.
+
+- [التوثيق](https://github.com/joyent/http-parser/)
+
+### c-ares
+
+بالنسبة لبعض طلبات الـ DNS الغير متزامنة ، يستخدم Node.js مكتبة C تسمى c-ares. يتم كشفها من خلال وحدة DNS في جافا سكريبت كأنها عائلة من دوال الحلول ( `resolve()`). تستخدم الدالة `lookup ()` ، وهي ما تستخدمه بقية النواة ، مكالمات المترابطة `getaddrinfo(3)` في libuv. والسبب في ذلك هو أن c-ares تدعم / etc / hosts ، و /etc/resolv.conf و /etc/svc.conf ، ولكن لا تفعل مع أشياء مثل mDNS.
+
+- [التوثيق](http://c-ares.haxx.se/docs.html)
+
+### OpenSSL
+
+يستخدم OpenSSL على نطاق واسع في كل من الوحدات النمطية `tls` و `crypto`. يوفر تنفيذ معركة الإختبار للعديد من وظائف التشفير التي يعتمد عليها الويب الحديث من أجل الأمان.
```

Comment:
_It provides battle-tested implementations_ is translated to:
_توفر العديد من وظائف التشفير التي تم اختبارها واقعيا_
the translation above seems like a bit "googlish" and has to be changed.

Reply:
yeah i use google in `OpenSSL` & `zlib` , because I wanted to compare my translation with the standard translation .....
Frankly, there is some word looks strange ...